extensions/wikia/AdEngine/js/slot/scrollHandler.js

```diff
@@ -9,21 +9,32 @@ define('ext.wikia.adEngine.slot.scrollHandler', [
     'use strict';
 
     var logGroup = 'ext.wikia.adEngine.slot.scrollHandler',
-        isRefreshed = {
-            PREFOOTER_LEFT_BOXAD: false,
-            PREFOOTER_RIGHT_BOXAD: false
-        },
-        reloadedView = {
-            PREFOOTER_LEFT_BOXAD: 0,
-            PREFOOTER_RIGHT_BOXAD: 0
-        };
+        isRefreshed = {},
+        reloadedView = {},
+        context = adContext.getContext(),
+        config = context.opts.scrollHandlerConfig ||
+            {
+                PREFOOTER_LEFT_BOXAD: { reloadedViewMax: 1 },
+                PREFOOTER_RIGHT_BOXAD: { reloadedViewMax: 3 }
+            };
 
     function init() {
-        if (adContext.getContext().opts.enableScrollHandler) {
+        if (context.opts.enableScrollHandler) {
+            for (var slotName in config) {
+                if (config.hasOwnProperty(slotName)) {
+                    isRefreshed[slotName] = false;
+                    reloadedView[slotName] = 0;
+                }
+            }
             win.addEventListener('scroll', adHelper.throttle(function () {
                 log('Scroll event listener has been added', 'debug', logGroup);
-                for (var slotName in isRefreshed) {
-                    if (isRefreshed.hasOwnProperty(slotName)) {
+                for (var slotName in config) {
+                    if (config.hasOwnProperty(slotName)) {
+                        if (config[slotName].hasOwnProperty('reloadedViewMax') &&
```

Comment:
I like the idea when missing `realoadedViewMax` param means `no limit`, but what about `-1` value? Consider updating condition here or JIRA ticket (strikethrough -1 part from AC?)